tests/MercadoPago/Client/Payment/PaymentRefundClientUnitTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace MercadoPago\Client\Payment;
+
+use MercadoPago\MercadoPagoConfig;
+use MercadoPago\Net\MPDefaultHttpClient;
+use MercadoPago\Client\Base\BaseClient;
+
+/**
+ * Payment Refund Client unit tests.
+ */
+final class PaymentRefundClientUnitTest extends BaseClient
+{
+    public function testCreateRefundSuccess(): void
+    {
+        $filepath = '../../../Resources/Mocks/Response/Payment/payment_refund.json';
+        $mockHttpRequest = $this->mockHttpRequest($filepath, 200);
```

Comment:
Revisar os testes que estão com variáveis no formato camel case
```suggestion
        $mock_http_request = $this->mockHttpRequest($filepath, 200);
```

Reply:
Ajustado!